cocos/platform/ohos/CCTextBitmap.cpp

```diff
@@ -0,0 +1,204 @@
+        layoutWidth = JSFunction::getFunction("StringUtils.getWidth").invoke<int>(text, fontSize, 400);
+    }
+    // typographic layout, setting maximum text width
+    OH_Drawing_TypographyLayout(cCTextBitmap->_typography, layoutWidth);
+    
+    // Obtains the maximum inherent width.
+    int realWidth = OH_Drawing_TypographyGetMaxIntrinsicWidth(cCTextBitmap->_typography);
+    // Obtaining the height
+    int realHeight = OH_Drawing_TypographyGetHeight(cCTextBitmap->_typography);
+    int textWidth = pWidth != 0 ? pWidth : realWidth;
+    int textHeight = pHeight != 0 ? pHeight : realHeight;
+
+    // Format used to describe the bit pixel, including color type and transparency type.
+    cCTextBitmap->_bitmap = OH_Drawing_BitmapCreate();
+    // COLOR_FORMAT_RGBA_8888：Each pixel is represented by a 32-bit quantity. 8 bits indicate transparency, 8 bits indicate red, 8 bits indicate green, and 8 bits indicate blue.
+    // ALPHA_FORMAT_OPAQUE：Bitmap has no transparency
+    OH_Drawing_BitmapFormat cFormat = {COLOR_FORMAT_RGBA_8888, ALPHA_FORMAT_OPAQUE};
+    
+    // Initializes the width and height of the bitmap object, and sets the pixel format for the bitmap.
+    OH_Drawing_BitmapBuild(cCTextBitmap->_bitmap, textWidth, textHeight , &cFormat);
+    
+    // Create a canvas object
+    cCTextBitmap->_canvas = OH_Drawing_CanvasCreate();
+    // Bind a bitmap object to the canvas so that the content drawn on the canvas is output to the bitmap (i.e., CPU rendering).
+    OH_Drawing_CanvasBind(cCTextBitmap->_canvas, cCTextBitmap->_bitmap);
+        
+    double xStart = calxStartPosition(align, layoutWidth, realWidth, textWidth);
+    double yStart = calyStartPosition((int)eAlignMask, realHeight, textHeight);
+    double position[2] = {xStart, yStart};
+    // Uses the specified color to clear the canvas. OH_Drawing_ColorSetArgb: Converts four variables (respectively describing transparency, red, green, and blue) to a 32-bit (ARGB) variable that describes colors.
+    OH_Drawing_CanvasClear(cCTextBitmap->_canvas, OH_Drawing_ColorSetArgb(0x00, 0xFF, 0x00, 0x00));
+    // Display Text
+    OH_Drawing_TypographyPaint(cCTextBitmap->_typography, cCTextBitmap->_canvas, position[0], position[1]);
+    
+    constexpr uint32_t stride = 4;
+    int32_t addrSize = pWidth * pHeight * stride;
+    // Obtains the pixel address of a specified bitmap. The pixel data of the bitmap can be obtained based on the pixel address.
+    cCTextBitmap->pixelAddr = OH_Drawing_BitmapGetPixels(cCTextBitmap->_bitmap);
+    cCTextBitmap->width = textWidth;
+    cCTextBitmap->height = textHeight;
+    return cCTextBitmap;
+}
+
+void* CCTextBitmap::getPixelAddr()
+{
+    return pixelAddr;
+}
+
+CCTextBitmap::~CCTextBitmap()
+{
+  OH_Drawing_CanvasDestroy(_canvas);
+//  OH_Drawing_BitmapDestroy(_bitmap);
```

Comment:
Doesn't `_bitmap` need to be released?

Reply:
the destructor function of CCTextBitmap seems never called. we fixed this bug